jubatus/core/anomaly/lof_storage.cpp

```diff
@@ -256,8 +256,18 @@ bool lof_storage::put_diff(const lof_table_t& mixed_diff) {
   for (lof_table_t::const_iterator it = mixed_diff.begin();
        it != mixed_diff.end(); ++it) {
     if (is_removed(it->second)) {
+      if (unlearner_) {
+        unlearner_->remove(it->first);
+      }
       lof_table_.erase(it->first);
     } else {
+      if (unlearner_) {
+        if (unlearner_->can_touch(it->first)) {
+          unlearner_->touch(it->first);
```

Comment:
This operation often create a new removed data (``kdist < 0``) in ``lof_table_diff_``.
But this information will not be propagate to other nodes, Because ``lof_table_diff_`` will be cleared in L.274.

We should create the removed data in ``lof_table_diff_`` after ``lof_table_diff_.clear()``.